lib/people.rb

```diff
@@ -434,6 +431,26 @@ def get_name_parts( name, no_last_name = false )
         last   = $3;
         parsed = true
         parse_type = 11;
```

Comment:
these rules should have different `parse_type` ids, and should have specific rspec tests